specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -1198,6 +1198,197 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NamespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SchemaGroupNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SchemaGroup"
+            },
+            "description": "Parameters supplied to create a Schema Group resource."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Schema Group successfully created.",
+            "schema": {
+              "$ref": "#/definitions/SchemaGroup"
+            }
+          },
+          "default": {
+            "description": "SchemaGroup error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SchemaGroups"
+        ],
+        "operationId": "SchemaGroups_Delete",
+        "x-ms-examples": {
+          "SchemaGroupDelete": {
+            "$ref": "./examples/SchemaGroups/EHSchemaGroupDelete.json"
+          }
+        },
+        "description": "Deletes an Schema Group from the specified Namespace and resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NamespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SchemaGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Schema Group successfully deleted."
+          },
+          "204": {
+            "description": "No content."
+          },
+          "default": {
+            "description": "SchemaGroup error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "SchemaGroups"
+        ],
+        "operationId": "SchemaGroups_Get",
+        "x-ms-examples": {
+          "SchemaGroupGet": {
+            "$ref": "./examples/SchemaGroups/EHSchemaGroupGet.json"
+          }
+        },
```

Comment:
There's an example reference but I don't see the file in the PR. Is that expected?

Reply:
Sorry, I seem to have missed this one. Let me see if it is worth adding an example to this at this moment.